### ISSUE TITLE
How to delete songs that already put into game

### ISSUE THREAD

**Issue by zhiames** [Scoolnik/SK.NRMods#4]
I put around 20 songs and realize that they all got pressed to one minute, and I had put a song that i didn’t want, is it possible to delete songs?

**Comment by Scoolnik** [comment]
Hi. Do you want to exclude default in-game songs from playlist?
For now there's no such an option, but I'll add it to requested features list.

**Comment by zhiames** [comment]
No I want to delete songs I’ve added to the game

**Comment by Scoolnik** [comment]
@zhiames Hi. Sorry for the late reply. You can just delete your songs from the folder you put it in and then restart the game 😁

**Comment by zhiames** [comment]
Yeah I deleted them successfully now cause initially I cannot delete them without running the game as it said can’t find the profile
